app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -317,6 +318,31 @@ class Settings(private val appContext: Context) : PreferencesHolder {
             numberOfAppLaunches > APP_LAUNCHES_TO_SHOW_DEFAULT_BROWSER_CARD
     }
 
+    /**
+     * Indicates if the sections Jump back in, Recently saved Recent explorations and Pocket,
+     * should be visible in the home screen.
+     */
+    private fun shouldShowMR2HomePageFeatures(): Boolean {
+        val experiments = appContext.components.analytics.experiments
+        val isExperimentBranch =
+            experiments.withExperiment(FeatureId.HOME_PAGE) { experimentBranch ->
+                (experimentBranch == ExperimentBranch.HOME_PAGE_TREATMENT)
+            }
+
+        return isExperimentBranch && numberOfAppLaunches > APP_LAUNCHES_TO_SHOW_HOME_PAGE_EXPERIMENTS &&
+         timeNowInMillis() - installationDate <= ONE_WEEK_MS
```

Comment:
Here we are making sure we are on the right experiment, the right branch and the user has less than week using the app after the second launch.

Reply:
@travis79 @jhugman is there an API for use to determine if the user it already enrolled in the experiment? I'm asking as after the user is enrolled in the experiment, the treatment should persist even if the 7 days period has already due.